pretyping/patternops.ml

```diff
@@ -416,8 +412,10 @@ let rec pat_of_raw metas vars = DAst.with_loc_val (fun ?loc -> function
       PLetIn (na, pat_of_raw metas vars c1,
                Option.map (pat_of_raw metas vars) t,
 	       pat_of_raw metas (na::vars) c2)
-  | GSort s ->
-      PSort s
+  | GSort GSProp -> PSort Sorts.InSProp
+  | GSort GProp -> PSort Sorts.InProp
+  | GSort GSet -> PSort Sorts.InSet
+  | GSort GType _ -> PSort Sorts.InType
```

Comment:
I think it would be better to export a `family` function for `glob_sort`, but this is not mandatory.